app/containers/message/Attachments.js

```diff
@@ -1,14 +1,40 @@
 import React from 'react';
 import { dequal } from 'dequal';
 import PropTypes from 'prop-types';
+import { Button, Text } from 'react-native';
 
 import Image from './Image';
 import Audio from './Audio';
 import Video from './Video';
 import Reply from './Reply';
+import RocketChat from '../../lib/rocketchat';
```

Comment:
@diegolmello, I tried to implement stories, But this **import** is not letting any stories(screen will be blank without any errors !!? ). **RocketChat** is imported to call the **sendMessage** function, so I tried to directly import the **sendMessage** function instead, which will make stories visible.

Reply:
@sumukhah, can you add the stories, after a fix that I did? Thanks for your contribution!